.github/workflows/unit.yml

```diff
@@ -17,6 +17,7 @@ jobs:
     name: 'unit'
     runs-on: '${{ matrix.os }}'
     strategy:
+      fail-fast: false
```

Comment:
This isn't related, but it was annoying to have a linux test fail and then not be able to see the Windows tests.